templates/workflows/run_conda-renv-module.yml

```
@@ -0,0 +1,69 @@
+# This is a workflow to run the {{ openscpca_module }} module
+#
+# Analysis modules are run based on three triggers:
+# - Manual trigger
+# - On pull requests where code in the module has changed
+# - As a reusable workflow called from a separate workflow which periodically runs all modules
+#
+# At initialization, only the manual trigger is active
+
+name: Run {{ openscpca_module }} analysis module
+env:
+  MODULE_PATH: analyses/{{ openscpca_module }}
+  AWS_DEFAULT_REGION: us-east-1
```

I don't know that it matters, but all of the OpenScPCA data is in `us-east-2` so we should probably use that. Which means this should probably be updated in the other templates too. 
```suggestion
  AWS_DEFAULT_REGION: us-east-2
```

---

noting I tacked this onto https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/529

---

Just kidding, I just went ahead and did it here: https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/556/commits/e0aaf61c1afa2b0722d8e46d5419df109cb3f321